components/AnimeHeader.js

```diff
@@ -16,11 +16,12 @@ const animeHeader = props => {
         </Link>
       </Fragment>
     ) : (
-      children
-    );
+        children
+      );
   };
 
   let stickyCover = props.router.pathname === "/" ? "relative" : "sticky";
+  const fallback = `/static/default-cover.jpg`
```

Comment:
This is better to move it out of the UI, however atm I'm not sure if the path is correct to render the default image. I think Dragon Ball Super doesn't have a cover image atm so that could be a good test